File: python/task3.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        for i in range(len(nums)-1, -1, -1):
            if (len(nums) > 1):
                if nums[i] == nums[i-1]:
                    nums.pop(i)

        return len(nums)

File: c++/algorithms/task3.cpp
#include <vector>

class Solution {
public:
    int removeDuplicates(std::vector<int>& nums) {
        int j = 1;

        for (int i = 1; i < nums.size(); i++){
            if (nums[i] != nums[i - 1]){
                nums[j] = nums[i];
                j++;
            }
        }

        return j;
    }
};